src/android/VerintXM.java

```diff
@@ -33,25 +33,25 @@
 /**
  * This class echoes a string called from JavaScript.
  */
-public class ForeSeeAPI extends CordovaPlugin {
+public class VerintXM extends CordovaPlugin {
 
     /**
-     * ForeSee API executor
+     * Verint API executor
      */
-    abstract class ForeSeeMethod {
+    abstract class VerintMethod {
         public abstract boolean invoke(JSONArray args, CallbackContext callbackContext, CordovaInterface cordova);
     }
 
     // Class tag for logs
     private final static String logTag = "CordovaVerintSDK";
```

Comment:
This might be better as 
```suggestion
    private final static String logTag = "CordovaVerintXM";
```